### ISSUE TITLE
[lecture_comparison][rob_markov_perf]long_equation_issue

### ISSUE THREAD

**Issue by sayaikegawa** [QuantEcon/lecture-python-advanced.myst#11]
In the case that equation is so long, equation reference in ```MyST``` does not show well like below:

Here is ```MyST```.
![Screen Shot 2021-01-22 at 1 02 23 pm](https://user-images.githubusercontent.com/44494439/105436186-dad10e80-5cb2-11eb-9926-31dd120c2ae9.png)

comparing to ```RST```
![Screen Shot 2021-01-22 at 1 02 35 pm](https://user-images.githubusercontent.com/44494439/105436220-f3d9bf80-5cb2-11eb-9b73-ff6786f055f6.png)

**Comment by mmcky** [comment]
@DrDrij it looks like we need floating `equation numbers` to support wide equations?

**Comment by mmcky** [comment]
@DrDrij any ideas on this one? Let me know when you might have some time to look at this. Thanks.

**Comment by mmcky** [comment]
@DrDrij I think I will break these equations across two lines.

**Comment by DrDrij** [comment]
@mmcky Agreed. It appears most solutions suggest reducing the length of the equation.
